_posts/2016-03-28-chime-quality-assurance.markdown

```diff
@@ -0,0 +1,47 @@
+
+As Bob V. stated in the last QA post, we have made a lot of changes. The last few years the QA department has accomplished a lot. What would take the QA team days to accomplish can now occur daily. In the past, if there were bugs, the release would have to wait until the next week because of the regression feedback loop. Now, for our newest product [Chime](https://hellochime.com) we deliver to prod multiple times a day, up to every 20 minutes for the varying underlying services. It is obviously not just QA's effort that has made this possible though; it has been a team effort across all departments (Development, Devops, and QA). Any company can accomplish continuous delivery if they have a determined team that can work well with one another.
+
+## Deciding what tools to use for Chime
+
+The newest Sittercity product, Chime, gave the QA team the opportunity to really work towards improving automation. There are many automation tools available. The foundational code a site uses will factor into what option is most appropriate. We primarily use ruby but also have a lot of different languages used in various projects. These may not work specifically for you, but they have met our needs so far: Watir, Cucumber, Appium, Applitools, and Jenkins.
+
+### Watir
+
+We chose to use [Watir](http://watir.com/) for the web automation. We have previously used it on our other [Sittercity](https://www.sittercity.com/) automation suites. It is easy to learn and very powerful in the right hands.
+
+### Cucumber
+Cucumber has many benefits for us. Since it's how we strive to structure our acceptance criteria. ♥ Cucumber and Watir-WebDriver sitting in a tree ♥ These two programs work really well with each other. It allows us to automate our web app and at the same time make the features easily readable to stakeholders, without any layers of abstraction.
+
+### Appium
+We have decided to try [Appium](appium.io) for our automated mobile testing. We use Calabash-iOS and Calabash-Android for our other mobile apps. However, we have ran into issues with them not being maintained for our needs in certain instances. This was why we decided to take a look at Appium. So far it's working well. We are looking forward to learning more about this tool and what opportunities it can provide for us.
+
+### Applitools
+
+Eyes_selenium is a gem by [Applitools](https://applitools.com/). This allows us to do automated UI testing. This allows us to test pixel and general layout level differences between pages. This is a great tool to use to make sure we don’t introduce any unpleasant visual bugs.
```

Comment:
Can you expand a bit on what makes Applitools different from Appium? From what I understand Appium is for testing on mobile devices and Applitools is for testing the web side. Is that right? I think adding another sentence to spell this out would be good.

If it can be used for mobile as well we should say that. I don't know enough about either one to tell you what is correct, though, so I'll trust your judgment.